src/screens/Payment/Checkout/checkout.jsx

```diff
@@ -1,29 +1,33 @@
-import {View, Text} from 'react-native';
-import React, {useEffect, useState} from 'react';
-import {styles} from './styles';
-import {CardForm} from '@stripe/stripe-react-native';
-import {Button} from '../../../components';
-import {initStripe} from '@stripe/stripe-react-native';
-import {useSelector} from 'react-redux';
+import React, {useCallback, useEffect, useState} from 'react';
+import {styles, Container} from './styles';
+import {initStripe, useStripe, CardForm} from '@stripe/stripe-react-native';
+import {Button} from '~src/components';
 
 const STRIPE_PUBLISHABLE_KEY = 'pk_test_4w4O2cKeqIBDIzucoUBDOKYO';
```

Comment:
look into how can we use environment variables in react native to store our keys instead of hardcoding them